lib/cli/commands/listpairs.ts

```diff
@@ -1,11 +1,34 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPairsRequest } from '../../proto/xudrpc_pb';
+import { ListPairsRequest, ListPairsResponse } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { getPairId } from '../../utils/utils';
+
+const HEADERS = [colors.red('PairId')];
+
+const formatPairs = (pairs: ListPairsResponse.AsObject): string[][] => {
+  const formatted: string[][] = [];
+  pairs.pairsList.forEach((pair) => {
+    const { baseCurrency, quoteCurrency } = getPairId(pair);
+    formatted.push([`${baseCurrency} / ${quoteCurrency}`]);
```

Comment:
It looks like all we are doing here after collapsing this into one column is putting a space before and after the `/` in the pair id, which seems unnecessary. I think we can just put the `pairId` in the output unformatted and keep it simple.

Reply:
You marked this resolved but I don't think anything changed and you didn't leave a comment. What are your thoughts? I still don't see the point of this and in looking around it seems most currency trading pair symbols are expressed without these spaces, eg `EUR/USD`.

In the future please don't mark as resolved making a change or leaving a comment for why you've resolved, it makes it harder to re-review a PR.